tasks/git-advanced.md

```diff
@@ -0,0 +1,140 @@
+# Advanced git
+**Enhance your git skills**
+
+> This task is optional.
+> You don't need it to advance further in the course.
+> However, you will definitely apply newly acquired skills
+> in your future work.
```

Comment:
It's not a block quote. It's better to leave pure paragraph.

Reply:
Agreed